LamestWebserver/LamestWebserver/HttpResponse.cs

```diff
@@ -67,9 +67,17 @@ public class HttpResponse
         /// <summary>
         /// Creates a new HttpResponse
         /// </summary>
-        public HttpResponse()
+        public HttpResponse(HttpRequest req)
         {
             Date = DateTime.Now.ToString(HtmlDateFormat);
+            if(req != null)
+            {
+                Range = req.Range;
+                if(Range != null)
+                Status = "206 Partial Content";
```

Comment:
I need this check because a range of null is the case if no range is requested and a request of null is the case if there is already a Problem with the request like a 404